python/pyspark/sql/tests/pipelines/test_graph_element_registry.py

```diff
@@ -0,0 +1,128 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+
+from pyspark.errors import PySparkException
+from pyspark.sql.pipelines.graph_element_registry import graph_element_registration_context
+from pyspark.sql import pipelines as sdp
+from pyspark.sql import SparkSession
+from pyspark.sql.tests.pipelines.local_graph_element_registry import LocalGraphElementRegistry
+
+
+class GraphElementRegistryTest(unittest.TestCase):
+    def test_graph_element_registry(self):
+        spark = SparkSession.builder.getOrCreate()
```

Comment:
why not reusing the ReusedSQLTestCase (for classic) and ReusedConnectTestCase (for connect)?